terraform/api_gateway_domain_name.tf.json.template.py

```diff
@@ -72,7 +72,25 @@
                         "evaluate_target_health": True,
                     }
                 }
-            }
-        } for lambda_name, gateway_id in gateway_ids.items() if gateway_id
+            },
+        },
+         **({
+             "aws_cloudwatch_log_group": {
+                 lambda_name: {
+                     "name": "/aws/apigateway/azul-%s-%s" % (lambda_name, config.deployment_stage),
+                     "retention_in_days": 90,
+                     "provisioner": [{
```

Comment:
Very elegant solution.

Reply:
I'm happy that it works, but I think it's hard to read / parse. I would prefer imperative statements here instead of one massive expression, but I figured that would be rejected since it goes against the style of the rest of the Terraform files. Let me know if you think it's worth changing.

Reply:
Also, `provisioner` can be single dict, too, it doesn't have to be list.